src/routes/event-match.tsx

```diff
@@ -91,28 +94,58 @@ const EventMatch = ({ eventKey, matchKey }: Props) => {
     [match],
   )
 
+  if (!isData(match)) {
+    return (
+      <Page
+        back={`/events/${eventKey}`}
+        name={
+          (m === null
+            ? matchKey + ' - '
+            : m.group + (m.num ? ' Match ' + m.num : '') + ' - ') +
+          (event ? event.name : eventKey)
+        }
+        class={matchStyle}
+      >
+        <Card class={matchStyle}>
```

Comment:
It is probably safer to create a new css block specifically for this, rather than reusing `matchStyle`. This branch doesn't have the redesigned match page styles yet, but if it did, I think it might break the layout of this card.